README.md

```diff
@@ -467,8 +467,8 @@ Python | 3.6 or higher | No | *N/A* | *Pre-installed or Installed by user* | [PS
 [GNU* FORTRAN Compiler](https://gcc.gnu.org/wiki/GFortran) | 7.4.0 or higher | Yes | apt | /usr/bin | [GNU General Public License, version 3](https://gcc.gnu.org/onlinedocs/gcc-7.5.0/gfortran/Copying.html)
 [Intel(R) oneAPI DPC++ Compiler](https://software.intel.com/en-us/oneapi/dpc-compiler) | latest | No | *N/A* | *Installed by user* | [End User License Agreement for the Intel(R) Software Development Products](https://software.intel.com/en-us/license/eula-for-intel-software-development-products)
 [hipSYCL](https://github.com/illuhad/hipSYCL/) | later than [2cfa530](https://github.com/illuhad/hipSYCL/commit/2cfa5303fd88b8f84e539b5bb6ed41e49c6d6118) | No | *N/A* | *Installed by user* | [BSD-2-Clause License ](https://github.com/illuhad/hipSYCL/blob/develop/LICENSE)
-[Intel project for LLVM* technology binary for x86 CPU](https://github.com/intel/llvm/releases) | Daily builds (experimental) tested with [20200331](https://github.com/intel/llvm/releases/download/20200331/dpcpp-compiler.tar.gz) | No | *N/A* | *Installed by user* | [Apache License v2](https://github.com/intel/llvm/blob/sycl/sycl/LICENSE.TXT)
-[Intel project for LLVM* technology source for NVIDIA GPU](https://github.com/intel/llvm/releases) | Daily source releases: tested with [20200421](https://github.com/intel/llvm/tree/20200421) | No | *N/A* | *Installed by user* | [Apache License v2](https://github.com/intel/llvm/blob/sycl/sycl/LICENSE.TXT)
+[Intel project for LLVM* technology binary for x86 CPU](https://github.com/intel/llvm/releases) | Daily builds before [dfe6e4e](https://github.com/intel/llvm/commit/dfe6e4e542ab363a7e77df7ccc069688daa5a51b), tested with [20230416](https://github.com/intel/llvm/releases/tag/sycl-nightly%2F20230416) | No | *N/A* | *Installed by user* | [Apache License v2](https://github.com/intel/llvm/blob/sycl/sycl/LICENSE.TXT)
+[Intel project for LLVM* technology source for NVIDIA GPU](https://github.com/intel/llvm) | Daily source releases, tested with [20230416](https://github.com/intel/llvm/tree/sycl-nightly/20230416) | No | *N/A* | *Installed by user* | [Apache License v2](https://github.com/intel/llvm/blob/sycl/sycl/LICENSE.TXT)
```

Comment:
Although not tested in CI, LLVM compiler should support AMD GPUs as well. Should we add that as well? "source for NVIDIA and AMD GPUs"

Reply:
Agreed. The logs using the 20230416 LLVM compiler with hip support are attached.
[rocblas_build_test_log.txt](https://github.com/oneapi-src/oneMKL/files/12969586/rocblas_build_test_log.txt): 100% tests passed, 0 tests failed out of 1893
[rocsolver_build_test_log.txt](https://github.com/oneapi-src/oneMKL/files/12969589/rocsolver_build_test_log.txt): 98% tests passed, 16 tests failed out of 797
[rocrand_build_test_log.txt](https://github.com/oneapi-src/oneMKL/files/12969588/rocrand_build_test_log.txt): 100% tests passed, 0 tests failed out of 85